articles/idam/idam.md

```diff
@@ -0,0 +1,83 @@
+---
+title: ID & Access Management API | UKCloud Ltd
+description: Provides information on managing permissions using the ID & Access Management API
+services: idam
+author: Simon Fish
+
+toc_rootlink: Getting Started
+toc_sub1:
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: IDAM
+toc_fullpath: Getting Started/idam.md
+toc_mdlink: idam.md
```

Comment:
Use correct file naming convention as detailed in https://docs.ukcloud.com/articles/other/other-ref-knowledge-guidelines.html#article-file-naming-conventions.
I'd suggest other-ref-idam.md
Make sure you make the change here, in the toc_fullpath metadata tag and in the actual name of the file.